tests/Behat/FeatureContext.php

```diff
@@ -298,4 +299,20 @@ public function assertVisibility($element): void {
     $this->assertVisuallyVisible($node);
   }
 
+  /**
+   * Create a dummy stakeholder.
+   *
+   * @todo Make it more flexible.
+   *
+   * @Given a dummy :name stakeholder
+   */
+  public function givenStakeholder($name) {
+    $stakeholder = Organisation::create([
```

Comment:
dont use variable call save directly on the return

Reply:
Previously fixed